SpatialGDK/Source/SpatialGDKEditorToolbar/Public/SpatialGDKEditorToolbar.h

```diff
@@ -85,9 +87,20 @@ class FSpatialGDKEditorToolbarModule : public IModuleInterface, public FTickable
 	void ShowSimulatedPlayerDeploymentDialog();
 	void OpenLaunchConfigurationEditor();
 
+	void BuildServerWorker();
+	void BuildClientWorker();
+	void BuildSimulatedPlayerWorker();
+	void BuildAll();
```

Comment:
Same with above.

Reply:
These items are cancellable from the notification dialog via the cancel button.